### DISCUSSION TITLE
Suggestion: Display the app version somewhere

### DISCUSSION THREAD

**Discussion by felipe19930** [cafali/SnapKey#35] [New Features & Ideas]
Can we have an easier way to verify what SnapKey version we're currently running? Here are my suggestions (and hopefully they make sense):

- Add the version string in the tooltip of the tray icon
- Add the version string as an unclickable context menu entry of the tray icon
- Add a "About" window

Thanks!

**Comment by cafali** [comment]
The version verification feature you suggested is now available in SnapKey 1.1.9.
